dali/kernels/imgproc/jpeg/jpeg_distortion_gpu_impl.cuh

```diff
@@ -16,20 +16,24 @@
 #define DALI_KERNELS_IMGPROC_JPEG_JPEG_DISTORTION_GPU_IMPL_CUH_
 
 #include <cuda_runtime_api.h>
-#include "dali/kernels/imgproc/jpeg/jpeg_distortion_gpu_kernel.h"
-#include "dali/kernels/common/block_setup.h"
-#include "dali/kernels/imgproc/surface.h"
-#include "dali/kernels/imgproc/sampler.h"
-#include "dali/kernels/imgproc/jpeg/dct_8x8_gpu.cuh"
-#include "dali/core/geom/vec.h"
-#include "dali/core/geom/mat.h"
 #include "dali/core/geom/box.h"
+#include "dali/core/geom/mat.h"
+#include "dali/core/geom/vec.h"
 #include "dali/core/util.h"
+#include "dali/kernels/common/block_setup.h"
+#include "dali/kernels/imgproc/jpeg/dct_8x8_gpu.cuh"
+#include "dali/kernels/imgproc/jpeg/jpeg_distortion_gpu_kernel.h"
+#include "dali/kernels/imgproc/sampler.h"
+#include "dali/kernels/imgproc/surface.h"
 
 namespace dali {
 namespace kernels {
 namespace jpeg {
 
+// TODO(janton): Purposedly not using the color space conversion utils here.
+// We need better color space conversion utilities and we are risking performance
+// using the conversion functions in color_space_conversion_impl.h
```

Comment:
What they are for in that case?

Reply:
They are used for the image decoder color conversion. Those utils can convert between types, so to allow for going from a narrower to a wider type and vice-versa we have to normalize to float first. This introduces extra multiplications that don't really make sense for the JPEG distortion kernel, since we are working with uint8. Making generic and very optimized color conversion utils it's beyond the scope of this PR, and should probably be tackled separately.

Reply:
Can you write this in the comment too?

Reply:
I went ahead and unify the color space conversion utils (we had 3 different ones)